corehq/ex-submodules/casexml/apps/phone/data_providers/case/load_testing.py

```diff
@@ -1,8 +1,16 @@
+from __future__ import annotations
```

Comment:
I'm not crazy about this, as I don't believe we have a tech-debt process in place here to remove these when we move to 3.10, or whenever this is slated to become the default.